docs/topics/mta-web-assessment-module.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+// * topics/mta-assessment-module.adoc
+
+:_content-type: REFERENCE
+[id="mta-assessment-module_{context}"]
```

Comment:
```suggestion
[id="mta-web-assessment-module_{context}"]
```
We had an issue with this ID this morning, so I am changing here

Remember you have included this in another PR, *BE VERY CAREFUL*.  It is awful to over-write changes with one PR that has issues.

For example, the image tags in MTA-1179 are fixed.